### PR TITLE
GT-519-mitigate-jakarta.el-dependency-vulnerability

### PR DESCRIPTION
**Pull request by vdeshkar**
Done with mitigating jakarta.el dependency for a2d2,
Done with build and test locally

### CODE REVIEW

File: a2d2-api/pom.xml [hunk 1]
```diff
@@ -53,12 +53,9 @@
 			<artifactId>xstream</artifactId>
 			<version>1.4.18</version>
 		</dependency>
+		<!-- <dependency> <groupId>com.fasterxml.jackson.core</groupId> <artifactId>jackson-databind</artifactId>
```
Comment (ddjain) [inline a2d2-api/pom.xml:56]: @vdeshkar please remove commented code.

File: a2d2-api/pom.xml [hunk 2]
```diff
@@ -336,16 +329,10 @@
...
-			<groupId>ch.qos.logback</groupId>
-			<artifactId>logback-classic</artifactId>
-			<version>1.2.3</version>
-		</dependency>
-		<dependency>
-			<groupId>ch.qos.logback</groupId>
-			<artifactId>logback-core</artifactId>
-			<version>1.2.3</version>
-		</dependency>
+		<!-- <dependency> <groupId>ch.qos.logback</groupId> <artifactId>logback-classic</artifactId>
```
Comment (ddjain) [inline a2d2-api/pom.xml:332]: @vdeshkar please remove commented code.

File: a2d2-api/pom.xml [hunk 3]
```diff
@@ -252,17 +253,9 @@
...
-			<artifactId>json-path</artifactId>
-			<version>2.4.0</version>
-			<exclusions>
-				<exclusion>
-					<groupId>org.ow2.asm</groupId>
-					<artifactId>asm</artifactId>
-				</exclusion>
-			</exclusions>
-		</dependency>
+		<!-- <dependency> <groupId>com.jayway.jsonpath</groupId> <artifactId>json-path</artifactId>
```
Comment (ddjain) [inline a2d2-api/pom.xml:256]: @vdeshkar please remove commented code.

**Review by ddjain — CHANGES_REQUESTED** [review]
Remove commented code

**Review by ddjain — DISMISSED** [review]

**Review by marianbuenosayres — DISMISSED** [review]

**Review by darshan-ht — APPROVED** [review]

**Review by darshan-ht — APPROVED** [review]

**Review by ddjain — APPROVED** [review]

**Pull request merged**